### PR TITLE
Feature/alternating neutral sorting

### LINKED ISSUE #30: As a user, when I click the neutral emoji, I see the articles sorted by alternating sentiment
- [ ] Add test

### PR DESCRIPTION
**Pull request by anthony-iacono**
### Features
- Closes #30 

This branch was originally intended to deliver more nuanced sorting when users click the neutral button. While white-boarding possible approaches, however, it occurred to me that dynamic sorting would be more useful than simply enhancing the sorting of a static list (we had talked about this in the beginning). 

With this branch, when the user opens an article to read (and we're assuming they read the article), `userSentiment` is updated to reflect that article's sentiment. And when the user returns to the feed, it is re-sorted based on the new `userSentiment`. 

PRO: This is a stronger feature than a static list that's sorted only once, because we can't expect the user to adhere to reading articles in that exact order. With dynamic sorting, however, the app will adapt to the user's choices and continuously nudge the user toward sentimentally balanced news consumption, which is our goal.

CON: The user may be disoriented when returning from having read an article to see the feed has been resorted. 

I can think of a number of reasons this would be disorientating. First, they may want to return to an article they previous read, but now have to scroll around to find it. Second, they may want to go back and read an article they had originally passed over and, again, have to scroll around to find it. 

To account for the first (and more likely) situation, we could quite easily add a 'history' feature. The naive implementation would be to remove articles from the feed as they are read and add them to a separate history page accessible through button/menu. 

### Refactors 
- Drilled `updateUserSentiment` down to the Cards
- Updated conditionals in `updateUserSentiment` to account for sentiment ranges rather than either just -1, 0, or 1

### Issues and Technical Debt
- None

### Other Comments, Concerns, or Questions
- Without checking truthiness of `userSentiment` in `updateUserSentiment`'s conditionals, it was throwing an error about `userSentiment` possibly being null. There were [other ways](https://bytenota.com/sovled-typescript-error-object-is-possibly-null-or-undefined/) to resolve the error, but this one seemed the least opaque.

### Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature work
- [ ] New and existing unit tests pass locally with my changes
- [ ] I ran Lighthouse and WAVE accessibility audits and fixed errors/warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/apiCalls.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ const checkResponse = (response: Response) => {
 export const getSentiment = (title: string, abstract: string): Promise<number> => {
   const text = title + ' ' + abstract;
   // add your API token here; remove before merging to main
-  const token = '';
+  const token = '' || '488823ba0f914b1f8eddc319191cbc7a';
```
Comment (nzambonivergara) [inline src/apiCalls.tsx:35]: Do you want to delete your token?

**Review by najinl — APPROVED** [review]

File: src/components/App/App.tsx [hunk 2]
```diff
@@ -38,8 +38,14 @@ const App = (): JSX.Element => {
...
   };
 
-  const updateUserSentiment = (userSentiment: number) => {
-    setUserSentiment(userSentiment);
+  const updateUserSentiment = (newUserSentiment: number) => {
+    let averageSentiment;
+    if (userSentiment) {
+      averageSentiment = (userSentiment + newUserSentiment) / 2;
+    }
+    console.log('userSentiment: ', userSentiment)
```
Comment (nzambonivergara) [inline src/components/App/App.tsx:46]: Remember to delete the console.logs :)

**Comment by najinl** [comment]
Thanks, looks great!

**Review by jphorner — APPROVED** [review]

File: src/components/App/App.tsx (hunk 2 above)
Comment (nzambonivergara) [reply src/components/App/App.tsx:46]: There's another one in line 26 of Feed.tsx.

**Review by nzambonivergara — APPROVED** [review]
Great job, @anthony-iacono! I really like our new way of sorting articles. We can talk more about the new sorting feeling a bit disorienting for users but that was not my experience.

**Pull request merged**